confmap/provider/eecprovider/README.md

```diff
@@ -0,0 +1,28 @@
+# EEC Provider
+
+This is an OpenTelemetry Collector `confmap.Provider` module that allows the
+Collector to be configured with the Dynatrace Extensions Execution Controller.
+
+> [!WARNING]
+> This is not intended for customer use, but is only intended for use by
+> Dynatrace when managing its own Collector deployments. As such, issues with
+> this component will fall outside our support policy.
```

Comment:
I think "only intended for use by Dynatrace when managing its own Collector deployments" makes it sound like it's only for use within Dynatrace the SaaS cluster or something like that. It's not clear that it's for use by Dynatrace in customer environments.

```suggestion
> This is not intended for direct customer use, but is only intended for use by
> Collector installations installed and managed by Dynatrace. Configuring this
> component directly is not supported.
```

Reply:
Yeah I wasn't really satisfied with my wording. I like yours better, thanks.